Dockerfile

```diff
@@ -5,8 +5,11 @@ FROM BASEIMAGE AS build
 COPY qemu-QEMUARCH-static /usr/bin/
 
 # device-mapper is needed for snapshot functionalities
+# openssl and wget needes to be updated.
 RUN apk add --no-cache \
-    device-mapper
+    device-mapper \
+    openssl \
+    wget
```

Comment:
This Dockerfile is for the ignite sandbox container image. Can you share what's the use case of wget in the sandbox?
If it's a problem in the OS images, we can do something similar in the OS images.

Reply:
![image](https://user-images.githubusercontent.com/25975144/166870946-4ea0abb3-3a0b-45cb-88b2-30faa62fa5f4.png)
In step 5, we use wget pull the firecracker image, and we get an error.

![image](https://user-images.githubusercontent.com/25975144/166872038-bbb0c45e-276d-4638-882d-4fd078de54ff.png)
And in the docker container, we can wget http but not https. Then we update wget by `apk add  wget` and wget works on https now.

So, it's actually a problem in the OS images.

Reply:
i have a program same here,but is a little different is the error code is 'time out',
so it's the build container can‘t connect the github.
i have a VPN in my pc , so change the network mode to "host" in the build container by recode the Makefile file. it's worked

```suggestion
 @@ -133,7 +134,8 @@ else
 endif
        $(DOCKER) build -t $(IMAGE):${IMAGE_DEV_TAG}-$(GOARCH) \
                --build-arg FIRECRACKER_VERSION=${FIRECRACKER_VERSION} \
-               --build-arg FIRECRACKER_ARCH_SUFFIX=${FIRECRACKER_ARCH_SUFFIX} bin/$(GOARCH)
+               --build-arg FIRECRACKER_ARCH_SUFFIX=${FIRECRACKER_ARCH_SUFFIX} bin/$(GOARCH)\
+               --network host

```
all in all, solve the network program. there are a lot of ways ,use the method you are most familiar with